import_ldraw.py

```diff
@@ -204,7 +211,7 @@ def parse_line(self, line):
         color = line[1]
 
         if color == '16':
-            color = self.colour
+            color = self.color
```

Comment:
Whoops, I just realized you already reverted the spellings back to the original form in your last commit. That's fine. don't change it again. :P Hopefully you understand what I meant, though. :)